<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Test"
           tests="5"
           failures="2"
           errors="0"
           skipped="0"
           time="39.424">
  <properties />
            <testcase name="loginChinese" time="7.513" classname="com.cases.Login">
                <failure type="java.lang.Throwable"
                  message="(null)"
              >
<![CDATA[java.lang.Throwable
  
报错信息:帐号密码为中文是否登录成功 expected [true] but found [false].
报错方法名:loginChinese(
报错类:Login
报错行号:55)
  com.cases.Login.loginChinese(Login.java:55)
]]>
      </failure>

          </testcase>
            <testcase name="loginErrorUser" time="8.443" classname="com.cases.Login">
                <failure type="java.lang.Throwable"
                  message="(null)"
              >
<![CDATA[java.lang.Throwable
  
报错信息:帐号对密码错误是否登录成功 expected [true] but found [false].
报错方法名:loginErrorUser(
报错类:Login
报错行号:34)
  com.cases.Login.loginErrorUser(Login.java:34)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="loginErrorNum" time="7.735" classname="com.cases.Login" />
                <testcase name="loginEmpty" time="7.646" classname="com.cases.Login" />
                <testcase name="loginConfim" time="8.087" classname="com.cases.Login" />
      
</testsuite>
